wp-content/themes/genesis-starter/gulp/config.js

```diff
@@ -58,8 +61,10 @@ export default {
       }
     },
     mode: {
+      inline: true,
       symbol: {
-        sprite: 'sprites.svg'
+        dest: '',
```

Comment:
The default is supposed to be the `<mode>`, in this case it would be `symbol`, but we are just provisioning for a single sprite, so I think it can be in the `public/images` folder.

If we actually want to support the creation of several sprites we need to configure this on a per project basis as it stands. Adding new sprite configurations and filters to the images task.